utils.go

```diff
@@ -397,10 +424,17 @@ func doesPluginSourceExist(gitURL string) (exists bool) {
 	return info.IsDir()
 }
 
-func getGoPath() (gopath string) {
-	if gopath = os.Getenv("GOPATH"); len(gopath) > 0 {
-		return
-	}
+var goPath = ""
```

Comment:
@djeebus we usually do globals at the top (if we can't avoid them)

Also, we try to do named return values 
`getGoPath() (goPath string, err error)`

Then set those values upon return

```
if len(cachedGoPath) > 0 {
   goPath = cachedGoPath
} else {
   // Parse goPath
   ...
   if err != nil {
      return
   }
}

return
```